docs/admin/administration.md

```diff
@@ -821,20 +821,20 @@ formatted again by running `dmg storage format`.
 To add a new server to an existing DAOS system, one should install:
 
 - the relevant certificates
-- the server yaml file pointing to the access points of the running
+- the server yml file pointing to the access points of the running
   DAOS system
 
 The daos\_control.yml file should also be updated to include the new DAOS server.
 
-Then starts the daos\_server via systemd and format the new server via
+Then start the daos\_server via the systemd command and format the new server via
```

Comment:
keeping original - please revert
(systemd is the facility ... the command would be systemctl)

Reply:
fixed